ios/SurveysUITests/Sources/Supports/AccessibilityIdentifier/AccessibilityIdentifier+Home.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AccessibilityIdentifier+Home.swift
+//  Surveys
+//
+//  Created by Su Ho on 07/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+extension AccessibilityIdentifier {
+
+    enum Home {
+
+        static let today = "today"
+        static let currentDate = "currentDate"
+        static let avatar = "currentDate"
```

Comment:
Hmm, copy-pasting issue? I'm wondering why your tests still run properly with duplicated view ids? 🤔 

```suggestion
        static let avatar = "avatar"
```

Reply:
Nice catch! 

FYI, In iOS, we can set the same accessibility id for different types of views. In this case, currentDate is a label and avatar is an image.

However, we can not set the same accessibility id for the same type. You will get this error.
![Screenshot 2022-11-10 at 10 31 38](https://user-images.githubusercontent.com/19943832/200994195-27c8d348-275a-44f1-8925-e7707a02e1ba.png)

Btw, I will update this :D

Reply:
@suho oh, thanks for clarifying 👌